downloader/src/main/kotlin/Main.kt

```diff
@@ -100,18 +98,6 @@ object Main {
         }
     }
 
-    private class DataEntityConverter : IStringConverter<DataEntity> {
-        override fun convert(name: String): DataEntity {
-            try {
-                return DataEntity.valueOf(name.toUpperCase())
-            } catch (e: IllegalArgumentException) {
-                e.showStackTrace()
-
-                throw ParameterException("Data entities must be contained in ${DataEntity.ALL}.")
```

Comment:
Does JCommander by default show the possible values if an invalid value was provided?
Btw, we need to improve `--help` to list the possible values for all enum parameters.

Reply:
Yes, it does:

    Exception in thread "main" com.beust.jcommander.ParameterException: Invalid value for --entities parameter. Allowed values:[PACKAGES, PROJECT]

Reply:
Nice.